doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,102 @@
+============================================
+Cross Validation
+============================================
+
+Cross validation is a type of model validation technique to estimate an algorithm's performance on unseen data.
+
+It essentially involves training the model on a subset of the data, and testing on the
+remaining data.
+
+The process is repeated for several complementary partitions and
+combines all the validation results generated (For instance, an average).
+
+One type of cross validation is known as :math:`k-` fold cross validation.
+
+Suppose we have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{N}`
+for a positive integer :math:`n` , then :math:`k-` fold cross validation partitions
+the data into :math:`P_1, P_2, ..., P_n` each with equal size (Where :math:`n=\frac{N}{k}`).
+
+Next, the model would be trained on :math:`P_1, P_2, ...,P_{n-1}` and is tested on :math:`P_n` to get performance measure
+:math:`a_1` .
+
+Then the model would train again from scratch on :math:`P_1, P_2, ...,P_{n-2}, P_{n}` and is tested on :math:`P_{n-1}`
+to get :math:`a_2`.
+
+This process is repeated until we have :math:`a_1, a_2, ..., a_n`, at which we can compute their mean and standard deviation.
```

Comment:
more grammar errors in here
replace "at which we can" with
"and then we can compute"

Reply:
Also you are between direct and indirect formulations.

"the model is trained" (indirect) and
"we train the model"

pls be consistent within this cookbook